plugins/uberdev/shared/code-fixer-output-v1.md

```
@@ -0,0 +1,72 @@
+## Code fixer output contract (v1)
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)